src/Helper/MyParcelCollection.php

```diff
@@ -712,7 +715,7 @@ private function apiEncodeReturnShipment($consignment)
      */
     private function getNewCollectionFromResult($result)
     {
-        $newCollection = new static();
+        $newCollection = new static;
```

Comment:
```suggestion
        $newCollection = new static();
```
When instantiating a new class, parentheses MUST always be present even when there are no arguments passed to the constructor.
https://www.php-fig.org/psr/psr-12/#4-classes-properties-and-methods

Reply:
Done